runtime/fiber.c

```diff
@@ -181,8 +181,7 @@ alloc_size_class_stack_noexc(mlsize_t wosize, int cache_bucket, value hval,
 #ifdef DEBUG
   stack->magic = 42;
 #endif
-  CAMLassert(Stack_high(stack) - Stack_base(stack) == wosize ||
-             Stack_high(stack) - Stack_base(stack) == wosize + 1);
+  CAMLassert(Stack_high(stack) - Stack_base(stack) >= wosize);
```

Comment:
I think that this should either:
- include a comment that explains why the test is weaker than one would naively expect
- or have a more precise test that makes it somewhat self-explanatory; for example, you could check that `high - base` is between `wosize` and `wosize + 15 / sizeof(value)`.

Reply:
Done, choosing the first possibility.